homeassistant/scripts/db_migrator.py

```diff
@@ -53,9 +47,13 @@ def print_progress(iteration: int, total: int, prefix: str='', suffix: str='',
         print("\n")
 
 
-def run(args) -> int:
+def run(script_args: List) -> int:
```

Comment:
Normally for arguments the typing should be Sequence instead of List.

The reason is that Sequence is more general and provides all you need from a typical list. (And List is issued when returning a list)